setup/command/apply.go

```diff
@@ -16,15 +16,24 @@ func (f *CommandFactory) Apply() cli.Command {
 				Name:  "quick",
 				Usage: "skips verification checks that normally run after 'terraform apply' has completed",
 			},
+			cli.BoolTFlag{
+				Name:  "push",
+				Usage: "setting it to false skips pushing local tfstate to s3 (default: true)",
+			},
 		},
 		Action: func(c *cli.Context) error {
 			args, err := extractArgs(c.Args(), "NAME")
 			if err != nil {
 				return err
 			}
 
+			provider, err := f.newAWSProviderHelper(c)
```

Comment:
I enquired with the chunnel team and can confirm that they ensure that aws env vars are set. Based on that information I believe it should not break their automation.